src/condor_utils/dag_parser.cpp

```diff
@@ -337,7 +337,7 @@ DagParser::ParseScript(DagLexer& details) {
 	while (true) {
 		std::string token = details.next();
 		if (DAG::SCRIPT_TYPES_MAP.contains(token)) {
-			cmd->SetType(DAG::SCRIPT_TYPES_MAP[token]);
+			cmd->SetType(DAG::SCRIPT_TYPES_MAP.at(token));
```

Comment:
`std::map::at` throws an exception if the element isn't found.  Can we do something like

```
const auto it = DAG::SCRIPT_TYPES_MAP.find(token);
if (it != DAG_SCRIPT_TYPES_MAP.end()) {
    cmd->SetType(it->second);
 ...
```
While it doesn't matter so much for this example, this way, there's no possibility to through an exception, and we only do the search of the map once.